UnityProject/Assets/Scripts/Objects/Construction/FloorDecals/FloorDecal.cs

```diff
@@ -22,6 +22,8 @@ public class FloorDecal : NetworkBehaviour
 
 		public bool isBlood = false;
 
+		public bool isFootprint = false;
```

Comment:
```suggestion
		public bool IsFootprint = false;
```
if you are going to make it public, then follow public fields naming convention-